{{cookiecutter.project_repo}}/Makefile

```
@@ -50,41 +50,44 @@ clean-test: ## remove test and coverage artifacts
 lint: checktypes checkstyle sast checklicenses ## run all checks
 
 checktypes: ## check types with mypy
-	mypy --ignore-missing-imports {{ cookiecutter.project_slug }} tests setup.py
+	mypy --ignore-missing-imports {{ cookiecutter.project_slug }} tests
```

This is something I missed on my original pass
```suggestion
	poetry run mypy --ignore-missing-imports {{ cookiecutter.project_slug }} tests
```

---

Done